packages/clarity-decode/src/data.ts

```diff
@@ -9,7 +9,7 @@ export function decode(tokens: Data.Token[]): DataEvent {
             let ping: Data.PingData = { gap: tokens[2] as number };
             return { time, event, data: ping };
         case Data.Event.Tag:
-            let tag: Data.TagData = { key: tokens[2] as string, value: tokens[3] as string[] };
+            let tag: Data.TagData = { tag: tokens[2] as string };
```

Comment:
Why no value any more?

Reply:
Like you said tags are not like normal tags :)